src/main/java/org/prebid/server/proto/openrtb/ext/request/rubicon/RubiconVideoParams.java

```diff
@@ -21,4 +21,6 @@
     Integer skip;
 
     Integer skipdelay;
+
```

Comment:
Same as above - we are adding Applogy bidder, so why are we modifying Rubicon bidder?